### ISSUE TITLE
Error deserializing /API/SERVICES

### ISSUE THREAD

**Issue by fabiolima1** [qJake/HADotNet#5]
Hello,

when I do
``
var _ServiceClient = ClientFactory.GetClient<ServiceClient>();
var Services = await _ServiceClient.GetServices();
``

I get this strange exception at BaseCliente.CS line 49:

` return JsonConvert.DeserializeObject<T>(resp.Content);`

_Unexpected character encountered while parsing value: {. Path '[7].services.apply.fields.entities.example', line 1, position 5643._

from newtonsoft.Json

The Json from HA is valid, but the library is having a problem parsing it.

I am using Home Assistant 0.107.6

this is the Json I got from HA.
```

[
   ...
   {
      "domain":"scene",
      "services":{
         "apply":{
            "description":"Activate a scene. Takes same data as the entities field from a single scene in the config.",
            "fields":{
               "entities":{
                  "description":"The entities and the state that they need to be.",
                  "example":{
                     "light.ceiling":{
                        "brightness":80,
                        "state":"on"
                     },
                     "light.kitchen":"on"
                  }
               }
            }
         },
         "create":{
            "description":"Creates a new scene.",
            "fields":{
               "entities":{
                  "description":"The entities to control with the scene.",
                  "example":{
                     "light.ceiling":{
                        "brightness":200,
                        "state":"on"
                     },
                     "light.tv_back_light":"on"
                  }
               },
               "scene_id":{
                  "description":"The entity_id of the new scene.",
                  "example":"all_lights"
               },
               "snapshot_entities":{
                  "description":"The entities of which a snapshot is to be taken",
                  "example":[
                     "light.ceiling",
                     "light.kitchen"
                  ]
               }
            }
         },
         "reload":{
            "description":"Reload the scene configuration",
            "fields":{

            }
         },
         "turn_on":{
            "description":"Activate a scene.",
            "fields":{
               "entity_id":{
                  "description":"Name(s) of scenes to turn on",
                  "example":"scene.romantic"
               }
            }
         }
      }
   }
   ...
]
```

**Comment by qJake** [comment]
(I updated your issue to just include the relevant JSON so it wasn't so long. 🙂)

This is interesting... obviously the API documentation from HA leaves much to be desired... so I had to make some guesses. In all of my services, the "Example" field is a string, but here it seems that the API is returning a JSON object as the example. I assume this is meant to be interpreted as-is and displayed as example JSON text.

I will need to update this line to handle a JSON object in addition to a string:

https://github.com/qJake/HADotNet/blob/f99c30a9325cee1aca75b49cd36eb14ad3f9dbf6/HADotNet.Core/Models/ServiceFieldObject.cs#L16
